src/styles/forms.scss

```diff
@@ -129,34 +129,12 @@
 			}
 		}
 
-		.FlySelect_Option {
-			height: 30px;
-			padding: 0 10px;
-
-			.FlySelect__SecondaryText {
-				margin-right: 0;
-			}
-
-			&:hover {
-				color: $white;
-				background: $green-dark50;
-
-				span {
-					color: $white;
-				}
-
-				svg path {
-					fill: $white;
-				}
-			}
-		}
-
 		.FlySelect_OptionGroup {
-			@include theme-background-gray5-else-graydarkalt;
 			height: 37px;
 			line-height: 37px;
 			text-align: left;
 			@include theme-color-gray-else-gray25;
+			@include theme-background-gray15-else-graydark50;
```

Comment:
Made darker to provide better contrast with striped/alternate rows option and corresponding styles.